pixeldata/src/lib.rs

```diff
@@ -532,17 +531,16 @@ impl DecodedPixelData<'_> {
 
     /// Retrieve object's rescale parameters.
     #[inline]
-    pub fn rescale(&self) -> Rescale {
-        Rescale {
-            intercept: self.rescale_intercept,
-            slope: self.rescale_slope,
-        }
+    pub fn rescale(&self) -> Vec<Rescale> {
```

Comment:
Maybe now we are better off storing `Rescale` values in `DecodedPixelData` itself (instead of separate `rescale_intercept` and `rescale_slope`), so that we can return a slice without making copies.